commit bd94cc11f8a813737052ba36e2bf63aa852ebe30
Author:     Aditya Bhatt <bhattadiCS@users.noreply.github.com>
AuthorDate: Thu Jul 16 11:05:33 2026 -0400
Commit:     Aditya Bhatt <bhattadiCS@users.noreply.github.com>
CommitDate: Thu Jul 16 11:05:33 2026 -0400

    docs: rename temperature plot to v3 to force markdown viewer cache refresh

diff --git a/research/generate_july_16_plots.py b/research/generate_july_16_plots.py
index 9de14b4..374ac0c 100644
--- a/research/generate_july_16_plots.py
+++ b/research/generate_july_16_plots.py
@@ -166,7 +166,7 @@ autolabel(rects3)
 
 
 plt.tight_layout()
-plt.savefig("ThesisDocs/images/temperature_drift_profile.png", dpi=300)
+plt.savefig("ThesisDocs/images/temperature_drift_profile_v3.png", dpi=300)
 plt.close()
 
 # -------------------------------------------------------------
